Added burger menu, applied auto formatting of vs-code

benszau opened the pull request:
Closes #3

shaardie commented on static/script.js:
```diff
@@ -0,0 +1,29 @@
+
+// Optional: Close menu when clicking outside
+document.addEventListener('click', function (event) {
+    const burger = document.querySelector('.burger');
```
Better use an `id` and `getElementById` here also.

benszau replied:
Done

shaardie commented on static/script.js:
```diff
@@ -0,0 +1,29 @@
+}
+
+// Optional: Close menu when clicking outside
+document.addEventListener('click', function (event) {
```
better only add this event when the menu is opened

benszau replied:
Done - I did note get the menu animation suppressed, but anyways

shaardie reviewed (changes requested):
The menu is missing in the desktop version and also on the upload site.

I would even argue to put it on every page, because it is irritating, if it's sometimes there and sometimes not.

benszau commented:
> The menu is missing in the desktop version and also on the upload site.
> 
> I would even argue to put it on every page, because it is irritating, if it's sometimes there and sometimes not.

I have moved the menu to a separate html file and added js code to load the menu code on the other pages

benszau commented:
> When going back, the menu is also still open

Done - I did note get the menu animation suppressed, but anyways

shaardie reviewed (approved):
Annika approves
